### DISCUSSION TITLE
Green Software Experts Program

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/discussions#21] [Ask Anything]
## What

Launching an ambassador program like the [Microsoft MVP](https://mvp.microsoft.com/) program of the [Google Developer Experts](https://developers.google.com/community/experts) program.

The bar should be set high, you should have to demonstrate **multiple** of the below.
- Speak about green software at conferences or at our green software meetups.
- Co-organize green software meetups.
- Create content on our website (or on other platforms) about green software.
- Contribute to our open source projects.
- Contribute to our community patterns/training projects.
- Contribute to any of the GSF projects.
- Further the field of green software in a series of categories which we can define.
- Complete our training, and any exam/certification we create in the future.

## Why

- This is a great way to recognize and reward people who are most active in the green software space.
MVP/GDE badges take effort to achieve but make you more employable, the same should be true for the GSE badge.
- If you are a company looking to hire someone to help make your systems more green, hiring a GSE should give you confidence in hiring.
- It's a great way to get more contributors for our projects, contributing to our projects is how you earn and maintain your GSE badge.
- gse.greensoftware.org could be a website that lists all the GSEs in the world, you can share your gse profile on social media and in your CV like gse.greensoftware.org/asim-hussain

## Background

We've had this idea boiling in the background since before the GSF was founded, in fact it was part of the GSF founding docs that we were going to launch an "Ambassador" program.

After year one we discussed the idea of launching the "Green Software Experts" program but we decided there really wasn't much we could measure people against on since we hadn't really released much by that point.

I think as we are reaching year 2 there is now plenty of material, lots of projects that need contributors, we've got training, patterns, SCI, plenty of open source projects and just generally the world has now evolved to the point where green software is an active field with lots of people taking part.

Thoughts?

**Comment by jawache** [comment]
Thoughts @aecurrie / @tmcclell ?

**Reply by tmcclell** [reply]
I love it!!

**Comment by aecurrie** [comment]
good idea!

**Comment by tmcclell** [comment]
- Asim - get the MVP person for EMEA come and discuss how their program is managed. Sarah asks on whether employees of the foundation or org can be awarded. Akshit - chair and co-chairs could take on a higher accreditation.

**Reply by adamj89** [reply]
I'd be happy to make this connection.

**Comment by navveenb** [comment]
Great idea !!
To add to Asim List - Contribute to GSF Patterns (additional points:)) , Implement SCI.

**Comment by alex-kroll** [comment]
Good idea!

**Comment by adamj89** [comment]
Suggestion that this is linked in part to measurable contributions linked to Asim's ideas above - this could help with automation of award/renewals.

**Comment by knsaravanan** [comment]
This is a fantastic initiative! Are there plans to release a next level exam after Green Software Practitioner?

**Comment by ijasanchez** [comment]
It is an excellent idea! Everyone involved in the software industry should know about green software. Promote the green software manifesto and principles (similar to https://manifesto.softwarecraftsmanship.org). The ambassador program is an opportunity for programmers and software engineers (with experience or in training) to learn how to contribute to our planet by building environmentally friendly software. 

In regions such as Latin America are necessary to know about green software, and an ambassador program would quickly promote this from spaces like universities or professional associations. In 2022, the second biggest revenue driver in the software market across Latin America was the application development software segment [[1]](https://www.statista.com/forecasts/1266664/segment-revenue-software-industry-latin-america-region).
